lib/gris/generators/templates/scaffold/db/seed.rb

```diff
@@ -0,0 +1,9 @@
+require './config/application' if File.file?('config/application.rb')
+
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rake db:seed.
+#
+# Examples:
+#
+#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
+#   Mayor.create(name: 'Emanuel', city: cities.first)
```

Comment:
New template to generate a blank db/seeds.rb file in newly scaffolded Gris client apps.